apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -555,3 +620,12 @@ NetSimPacketEditor.prototype.updateBitCounter = function () {
 NetSimPacketEditor.prototype.onRemovePacketButtonClick_ = function () {
   this.removePacketCallback_(this);
 };
+
+/**
+ * Remove the first bit of the packet binary, used when sending one bit
+ * at a time.
+ */
+NetSimPacketEditor.prototype.consumeFirstBit = function () {
+  this.setPacketBinary(this.getPacketBinary().substr(1));
```

Comment:
Not unless you feel like this method should also return the bit it consumes.

* `getFirstBit` reads and returns the first bit of the packet without modifying the packet
* `consumeFirstBit` removes the first bit of the packet, but returns nothing.

I separated these steps because I need to read the first bit before I try to set the wire state, but I didn't want to actually consume the bit until I knew setting the wire state had succeeded.